docs/security/faq.md

```diff
@@ -83,7 +83,7 @@ No service can access data of other services except when it had to be allowed -
 Groups are also used for controlling access.
 This is very flexible as you can give any user/service access to more things by adding them to appropriate group.
 
-## Aren't users unsecure? What are the limitation of this design?
+## Aren't users insecure? What are the limitation of this design?
```

Comment:
"unsecure" is correct term here as in non-secured. "insecure" is property of a person who is not confident about themselves.

Reply:
```suggestion
## Aren't users unsecure? What are the limitation of this design?
```